client/src/components/container/user/EventPage/index.js

```diff
@@ -1,27 +1,50 @@
 import React, { Component } from 'react'
-import { Button } from "antd";
+import { Button ,Alert} from "antd";
 
 import EventPageContent from '../EventPageContent/EventPageContent'
 import PopupBtnEventcomp from '../PopupBtnEventcomp/PopupBtnEventcomp'
+import Axios from 'axios';
 
 import './style.css'
 
 class EventPage extends Component {
-
-  render() {
+  state={
+    image:"",
+    title:"",
+    otherEventProps:[],
```

Comment:
otherEventProps will be an object not an array.
```suggestion
    otherEventProps: null,
```